roles/base/tasks/main.yml

```
@@ -18,6 +39,7 @@
     - librbd-dev
     - lshw
   when: ansible_os_family == "Debian"
+  ignore_errors: yes
```

Why do we we ignore errors?